Create ui-c8y-1018-503-66-the-app-logo-height-was-not-correctly-set-if-a-new.md (#1814)
SoftwareAG/c8y-docs: content/change-logs/ui-c8y-1018-503-66-the-app-logo-height-was-not-correctly-set-if-a-new.md

---
date: ""

> title: App logo height default value corrected when creating new branding

Reviewer: ```suggestion
title: App logo height set properly when creating new branding
```

> title: The app logo height was not correctly set if a new branding is created [GRAFT][release/y2024] (#5843)

Author: The Claude AI suggested to use the following title:
```suggestion
title: App logo height fixed when creating new branding
```

product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-0UgqXH1Ys
    label: Administration
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-52036
version: 1018.503.66
---

> The app logo height was not correctly set if a new branding is created [GRAFT][release/y2024] (#5843)

Author: The Claude AI suggested to use the following body:
```suggestion
Fixed an issue where the app logo height was not set correctly when creating a new branding. The app logo height will now be set properly when generating a new branding.
```
